cmd/skipper/listflag.go

```diff
@@ -36,16 +36,40 @@ func (lf *listFlag) Set(value string) error {
 
 	lf.value = value
 	lf.values = strings.Split(value, lf.sep)
+
+	if err := lf.hasOnlyAllowedValues(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (lf *listFlag) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var values []string
+	if err := unmarshal(&values); err != nil {
+		return err
+	}
+
+	lf.value = strings.Join(values, lf.sep)
+	lf.values = values
+
+	if err := lf.hasOnlyAllowedValues(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (lf *listFlag) hasOnlyAllowedValues() error {
```

Comment:
The name is not good, because functions with `has` prefix should return bool.

Reply:
what about rename to `validate() error`?

Reply:
fine for me